Sources/BraintreeShopperInsights/BTShopperInsightsClient.swift

```diff
@@ -17,6 +17,9 @@ public class BTShopperInsightsClient {
     // MARK: - Private Properties
     
     private let apiClient: BTAPIClient
+
+    /// This value should be the shopper session ID returned from your server SDK request
+    private let shopperSessionID: String? = ""
```

Comment:
We instead want to add this to the initializer for this class as a required property so merchants will set it when initializing the shopper insights client. Then in our analytics we can set this property when making the analytics request.